apps/protocol-frontend/src/hooks/useUserActivityTypesList.ts

```diff
@@ -0,0 +1,14 @@
+
+import { useUser } from '../contexts/UserContext';
+import { useQuery } from '@tanstack/react-query';
+import { UIActivityType } from '@govrn/ui-types';
+
+export const useUserActivityTypesList = () => {
+  const { govrnProtocol: govrn } = useUser()
+  const { isLoading, isFetching, isError, error, data } = useQuery(['activityTypes'], async (): Promise<UIActivityType[]> => {
+    const data = await govrn.custom.listActivityTypesByUser({})
```

Comment:
Shouldn't the hook take in filter args that resemble listActivityTypesByUser so we don't have to create a new hook for different args?

Reply:
Ah, good idea! To make sure I'm on the same page -- we probably want to ensure that the hook's args match the available args on the `listActivityTypesByUser` so that we'd be able to cover all possible use cases in the future? That makes a lot of sense and would be a good pattern for creating these hooks moving forward. This one in particular didn't utilize the args but that's a good pattern to follow for the ones that do. 

Makes sense to support this at a structural level.